docs/userman/gui/extension/dynamic_element/list_of_value_props.md

```diff
@@ -0,0 +1,159 @@
+The previous section on [Tabular properties](tabular_data_props.md) explains creating dynamic elements for tabular data values.
+For collections like lists, a more complex setup is needed to support dynamically updating arrays. In Taipy GUI, lists
+can bind to Python variables or expressions, instantly reflecting changes in the UI. Custom elements handling lists must
+declare their properties, similar to tabular properties, using the PropertyType class to define array formats. This allows
+multidimensional data binding, so any changes to the list refresh the UI display.
+
+# Using list of values
+
+In this section, we will expand the dynamic element library by adding a new dynamic custom element.
+
+This dynamic element will accept a property containing a list of values and display it within a list. When a Python
+variable is bound to this property, updates to the variable will immediately reflect in the list content shown on the
+page, ensuring real-time synchronization.
+
+## Declaring a dynamic element {data-source="gui:doc/extension/example_library/example_library.py#L45"}
+
+Starting from the code mentioned above, here is how you would declare this new element:
+```py   title="example_library.py"
+from taipy.gui.extension import Element, ElementLibrary, ElementProperty, PropertyType
+
+class ExampleLibrary(ElementLibrary):
+    def __init__(self) -> None:
+        # Initialize the set of visual elements for this extension library
+        self.elements = {
+            "visual_label_list": Element(
+                "lov",
+                {
+                    "lov": ElementProperty(PropertyType.lov),
+                    "sort": ElementProperty(PropertyType.string),
+                },
+                # The name of the React component (VisualLabelList) that implements this custom
+                # element, exported as LabeledItemList in front-end/src/index.ts
+                react_component="VisualLabelList",
+            )
+        }
+```
+The declaration of this dynamic element is very similar to what we created in the
+[Tabular properties](tabular_data_props.md). The detailed explanation of the code is as follows:
+
+- The *visual_label_list* element includes two properties: *lov* and *sort*.
+- The *lov* property has the type *PropertyType.lov*, meaning it holds a list of value and is dynamic.
+- The *sort* property has the type *PropertyType.string*, meaning it holds a string value. Because this property is
+  static, it would nevertheless not reflect changes in the bound variable values.
+- The *get_name()* method in the *ExampleLibrary* class returns the name of the library as a string. This name is used
```

Comment:
You don't need to repeat these.
get_name() and get_elements() were already explained in the sections before.

You may want to say: "get_name() and get_elements() remain the same as in the previous stages of building the extension library example."